docs/3-getting-started.md

```diff
@@ -4,7 +4,7 @@ Welcome to the documentation about Zarf, the air-gap tool! Let's get you started
 
 ## Installing Zarf
 
-There are multiple ways to get the Zarf CLI onto your machine including installing from the Defense Unicorns Homebrew Tap, downloading a prebuilt binary from our GitHub releases, or even building the CLI from scratch yourself.
+There are multiple ways to get the Zarf CLI onto your machine including installing the Defense Unicorns Homebrew Tap, downloading a prebuilt binary from our GitHub releases, or even building the CLI from scratch yourself.
```

Comment:
Can you turn this into a list as well  (with links if applicable)
- defense Unicorns Homebrew tap
- downloading prebuilt binary from Github release
- building the CLI from source

Reply:
Done with included links